### PR TITLE
Created leaderboard structure

### PR DESCRIPTION
**Pull request by JOdiemo**
Created the leaderboard using the given wireframe
Added CSS to the project
Updated Linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,40 +29,22 @@
...
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+1. Clone the repository: https://github.com/JOdiemo/leaderboard/tree/leaderboard-final
+2. Copy to your local disc: you@your-Pc-name:~$ cd <folder>
```
Comment (mckent05) [inline README.md:33]: - In order to make your repo look more professional I'd suggest that you include steps on how to run this project locally 👍 👍 .

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @JOdiemo,

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Your project looks good 👌 
- Your code is clean ✔️ ✔️ 
- Descriptive PR summary and title 👍 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md:33]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Jorum,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Project is set up correctly
- Webpack is correctly configured

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor. 
- Generally, the subheading text Author1 should be replaced with your full name in your ReadMe.
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**